cognite/neat/_session/_mapping.py

```diff
@@ -1,28 +1,44 @@
 from datetime import datetime, timezone
 
+from cognite.neat._client import NeatClient
+from cognite.neat._constants import DEFAULT_NAMESPACE
+from cognite.neat._rules.importers import DMSImporter
+from cognite.neat._rules.models.dms import DMSValidation
 from cognite.neat._rules.models.mapping import load_classic_to_core_mapping
-from cognite.neat._rules.transformers import RuleMapper
+from cognite.neat._rules.transformers import AsParentName, RuleMapper, VerifyDMSRules
+from cognite.neat._store._provenance import Agent as ProvenanceAgent
 from cognite.neat._store._provenance import Change
 
 from ._state import SessionState
-from .exceptions import session_class_wrapper
+from .exceptions import NeatSessionError, session_class_wrapper
 
 
 @session_class_wrapper
 class MappingAPI:
```

Comment:
to keep our interfaces/interaction with NeatSession uniform, MappingAPI should have split among mappings done on instances and mappings done on data model: